src/visit_vtk/lightweight/vtkDataSetFromVolume.h

```diff
@@ -163,7 +163,7 @@ class VISIT_VTK_LIGHT_API EdgeHashTable
   public:
                       vtkDataSetFromVolume(vtkIdType ptSizeGuess);
                       vtkDataSetFromVolume(vtkIdType nPts, vtkIdType ptSizeGuess);
-    virtual          ~vtkDataSetFromVolume() { ; };
+    virtual          ~vtkDataSetFromVolume() { ; }
```

Comment:
Why do we need this empty implementation of the destructor as opposed to just having the default destructor? Is it maybe to support the `virtual` keyword?

Reply:
Destructors need to be declared virtual.  Not sure what happens if it isn't declared at all, IIRC it is needed to fully declare the class

Reply:
But why declare with `{ ; }`? Is that necessary? I guess it was that way before.